app/controllers/api/v1/registrations/registrations_controller.rb

```diff
@@ -35,23 +33,27 @@ def show
 
   def create
     # Currently we only have one lane
+    # QUESTION: Should we be calling registration_params[:competing]?
```

Comment:
we could, but the I think we can just remove `registration_params` as this is going to be dynamic depending on the lanes used